tlf-disposition.Rmd

```diff
@@ -107,14 +110,16 @@ n_reason
 In the code below, we calculate the number and percentage of participants who complete the study.
 We split `n_reason` because we need to customize the row order of the table.
 
+
 ```{r}
-n_complete <- n_reason %>% filter(row == "Completed")
+n_complete <- n_reason %>% 
+  filter(row == "Completed")
 
 n_complete
 ```
 
 In the code below, we calculate the number and percentage of participants who discontinued the study for different reasons.
-Here we use `paste0("    ", row)` to add some leading space for individual reasons for study discontinuation.
+For display purposes, we use `paste0("    ", row)` to add leading spaces to mimick identation in the final report.
```

Comment:
For display purposes,  `paste0(" ", row)` is used to add leading spaces to mimick identation in the final report

Reply:
```suggestion
For display purposes, `paste0("    ", row)` is used to add leading spaces to mimick identation in the final report.
```